utils/decode_kallsyms.py

```diff
@@ -0,0 +1,53 @@
+import bisect
+import re
+
+
+def kallsyms_search(kallsyms_lines: list[str], address: int):
+    # kallsyms lines looks like this
+    #
+    # ffffffffaa88e2f0 T kfree
+    # ffffffffaa88e400 T __ksize
+    # ffffffffaa88e530 T ksize
+    def key(line): return int(line.split()[0], 16) - 1  # pylint: disable=multiple-statements
```

Comment:
Actually, there is the `lambda` built-in that allows to do one-liner functions:

```
key = lambda line: int(line.split()[0], 16) - 1
```

Also, it won't require suppressing of pylint rules.

Reply:
It started as a lambda, then @fgelcer changed it to a function, and now back to a lambda. I wonder what's that about.

Reply:
yes, from PEP8 PoV you were right that a lambda like it is would be an error, but we decided to ignore it...
funtionality wise, it does what is expected, so if you are not strongly against it, please approve